vmdk_plugin/drivers/photon/photon_driver.go

```diff
@@ -577,6 +580,9 @@ func (d *VolumeDriver) Mount(r volume.MountRequest) volume.Response {
 func (d *VolumeDriver) Unmount(r volume.UnmountRequest) volume.Response {
 	log.WithFields(log.Fields{"name": r.Name}).Info("Unmounting Volume ")
 
+	// dirty the background refcounting process if any
+	d.refCounts.MarkDirty()
+
 	// if the volume is still used by other containers, just return OK
```

Comment:
Mark dirty will mark the refcount reattempt dirty. So that the background refcounting breaks if it was in process.
So just mark it dirty and proceed.

lines 596-600 check for a completely separate flag about refcounting in whole has been successful or not.